client/state/geo/selectors.js

```diff
@@ -32,3 +32,13 @@ export function getGeo( state ) {
 export function getGeoCountry( state ) {
 	return get( getGeo( state ), 'country_long', null );
 }
+
+/**
+ * Returns the current browser IP geolocation abbreviated country name.
+ *
+ * @param  {Object}  state Global state tree
+ * @return {?String}       Current browser IP geolocation short country name
+ */
+export function getGeoCountryShort( state ) {
```

Comment:
New selectors are now added to the `state/selectors` folder. This allows us to more easily import selectors and search for them in devdocs. See also: https://github.com/Automattic/wp-calypso/blob/master/client/state/selectors/README.md